regexes/client/browsers.yml

```diff
@@ -515,6 +515,27 @@
   name: 'Baidu Spark'
   version: '$1'
 
+# Yandex Browser (alpha)
+- regex: 'YaBrowser(?:/(\d+[\.\d]*)) \(alpha\)?'
```

Comment:
Is the last `?` on the right position?

Reply:
Yes, is based on the regex for Yandex Browser.

Reply:
Should the regex not be `'YaBrowser(?:/(\d+[\.\d]*))? \(alpha\)'` ? The last `?` in the Yandox browser regex makes the version part optional.